system/Database/Forge.php

```diff
@@ -429,8 +491,12 @@ protected function _createTable($table, $if_not_exists, $attributes)
 				: "\n\t".$this->_processColumn($columns[$i]);
 		}
 
-		$columns = implode(',', $columns)
-			.$this->_processPrimaryKeys($table);
+                
+		$columns = implode(',', $columns);
+                
+                $columns .= $this->_processPrimaryKeys($table);
```

Comment:
Just removing the _ should be good: `processPrimaryKeys`. I didn't see a method name that would conflict but just did a quick search through the changes.